source/includes/_reserve_page_archive.md

```diff
@@ -0,0 +1,353 @@
+# Reserve Page Archive
+> **Definition**
+
+```
+GET https://demo.ticketswitch.com/f13/reserve_page_archive.v1
+```
+
+This API call should be used in preference to the [status](#status) API call.
+It is faster than the status API call.
+It returns an exact replica of the original reservation response.
+This means that you can access all that data that you would normally require from a status call, except that the status and the amount of time left on the reservation will not have been updated (as it will be exactly the same as the original reservation response).
+The reservation responses are cached for 24 hours.
```

Comment:
Might be good to state what will happen if a request is made for a reservation after the 24 hours have elapsed